python/pyarrow/_csv.pyx

```diff
@@ -34,8 +34,8 @@ from pyarrow.lib cimport (check_status, Field, MemoryPool, Schema,
                           pyarrow_unwrap_batch, pyarrow_unwrap_table,
                           pyarrow_wrap_schema, pyarrow_wrap_table,
                           pyarrow_wrap_data_type, pyarrow_unwrap_data_type,
-                          Table, RecordBatch)
-from pyarrow.lib import frombytes, tobytes
+                          Table, RecordBatch, StopToken)
```

Comment:
Is there a JIRA to expose `StopSource` to python at some point?  There are reasons other than signals someone may want to cancel an operation.  For example, a GUI-based application may have a cancel button.  A web server may want to cancel if the TCP connection for some analysis request is lost.

Reply:
No JIRA yet. We can open one, or do it when needed.